CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Extensions/UIView.swift

```diff
@@ -6,7 +6,7 @@
 //
 
 import UIKit
-extension UIView {
+public extension UIView {
```

Comment:
[Request] I am worried making extensions to UIView public. It might clash with client app's UIView extensions. Also the method `loadNib()` uses `giniCaptureBundle()`. 

As these are helper convenience methods I would rather we duplicate the methods that are needed in Bank SDK.

What do you think? Is there a better option?